pkg/module/security_scan.go

```diff
@@ -97,7 +97,7 @@ func appendProtecodeImagesLayers(descriptor *ocm.ComponentDescriptor, config *Se
 					Labels:  []ocmv1.Label{*imageTypeLabel},
 					Version: imageTag,
 				},
-				Type:     ociimage.TYPE,
+				Type:     itociartifact.LEGACY_TYPE, //#561
```

Comment:
what's the meaning of this `#561`? a PR number? do we really need this in the comment?

Reply:
Yes, PR number from OCM library that introduced that change. Leftover from investigating, removed.